src/excel_adapter/excel_adapter.go

```diff
@@ -43,18 +46,30 @@ func getIP(sheet string, document internalModels.Document) string {
 	panic(fmt.Sprintf("excel adapter: getIP: Missing board %s IP\n", sheet))
 }
 
-func AddExpandedPackets(document internalModels.Document, objects ...models.FromBoards) {
-	globalInfo := models.GlobalInfo{BoardToIP: getBoardToIPs(document), UnitToOperations: make(map[string]string)}
+func Update(document internalModels.Document, objects ...models.FromDocument) {
+	globalInfo := getGlobalInfo(document)
```

Comment:
Put in separate `addGlobalInfo` function

Reply:
We are splitting the code too much I think by doing so